how-we-do-hiring-and-hr-at-posthog.md

```diff
@@ -0,0 +1,112 @@
+title: How We Do Hiring and HR at PostHog
+author: mo-shehu
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+Our main selling point as a company is transparency. Being an [open-source company](https://posthog.com/blog/open-source-eating-saas), this isn’t surprising: our code, designs, and content are all largely available [online](https://github.com/posthog), for free, even while still a work in progress. This transparency also extends to HR functions from our salary ranges to our employment processes - all of which you can view in our [company handbook](https://posthog.com/handbook/company/story). Such openness has attracted candidates from all over the world who believe in and want to build on this level of transparency.
+
+As of writing, a third of our staff is based in the US, another third in the UK, and the rest spread out across the globe in places like South Africa, Brazil, the Ukraine, Congo, and Russia. This is deliberate, as diverse backgrounds contribute new viewpoints, generate fresh ideas, and unearth new ways of solving problems with our product, marketing, and community-building.
+
+## How we hire
+
+[Charles](https://posthog.com/handbook/company/team#charles-cook-business-operations), our Business Head, is famously enthusiastic about admin, and has done a fantastic job of managing our systems so far. These include our payroll, leave system, share grants, IT onboarding, and other critical tasks that ensure a smooth onboarding experience for each new candidate. [Eltje](https://posthog.com/handbook/company/team#eltje-lange-people-and-talent) took over the HR function and now sets up meetings with candidates, guides them through the interview process, and sets up their benefits and perks.
+
+It takes about 2 weeks to go from receiving a candidate’s application to sending them an offer letter. This timeline may vary based on the time zones they’re in, how many people on our team they need to meet, whether or not we already have salary benchmarks in place for their country, and other factors.
+
+### Job descriptions
+
+For every new role, Eltje works with the hiring manager to generate a job description that closely describes what the role will entail. As an early-stage startup, roles at PostHog are fluid and can evolve over time, and our job descriptions are more of a wishlist than a set of directives. We then run the job description through a gender decoder (like [this one](https://www.totaljobs.com/insidejob/gender-bias-decoder/) from Totaljobs) which analyses the words and suggests non-gendered alternatives where necessary. In an industry like ours, it’s easy to attract (and write JDs for) predominantly white males, but part of our DNA is about encouraging diversity of thought, life experiences, and backgrounds. We also tweak the ‘locations’ of each role for every hiring round, partially for diversity’s sake, but mainly because we’re a fully-remote company and don’t actually have offices anywhere.
```

Comment:
I usually use [this](http://gender-decoder.katmatfield.com/) gender decoder, but any is fine really

Reply:
Saw that one as well when I Googled decoders, but the one I posted has a better/more appealing UI so the reader might find it easier to use.